pkgs/development/python-modules/youtube-dlc/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "youtube_dlc";
+  version = "2020.11.11.post3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "WqoKpfvVPZrN+pW6s8JoApJusn5CXyPcg9VcsY8R0FM=";
+  };
+
+  outputs = [ "out" "dev" ];
```

Comment:
```suggestion
```

common for python packages to use this to pass runtime packages
```
cat ./result-dev/nix-support/propagated-build-inputs
/nix/store/18656kvqazm74bj7k3mdkwmdlqfyf581-python3-3.8.6  /nix/store/wwsf5czpcmxbqzqv5p623cc40jdn6cg0-python3.8-youtube_dlc-2020.11.11.post3
```

Reply:
That's solved.